apps/cookbook/src/app/showcase/checkbox-showcase/checkbox-showcase.component.html

```diff
@@ -1,8 +1,10 @@
 <p>
-  A Checkbox is a control that permits the user to select options. This can be a binary choice, i.e.
-  a choice between one of two possible mutually exclusive options. For example, the user may have to
-  answer 'yes' (checked) or 'no' (not checked) on a simple yes/no question. Multiple check boxes in
-  a group may be used to either select multipe items.
+  A checkbox is a control that allows the user to select options. This can represent a binary
+  choice, i.e., a choice between two mutually exclusive options. For example, the user may answer
+  'yes' (checked) or 'no' (not checked) to a simple yes/no question. Multiple checkboxes can be used
+  to select several items in a list. In cases where there is a long list of selectable items, a
+  "Select all" option at the top of the list can be added to enhance usability. If a user deselects
+  an item, the checkbox for "Select all" should display the state "Partially selected."
```

Comment:
Now that I see this added to the description, I feel it would be better if this has its own example instead. 

Could you duplicate the 'Used in a list' example, add the partially checked logic and place this description in the example instead?